src/module-elasticsuite-thesaurus/etc/logging.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<logging xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Logging:etc/logging.xsd">
+    <groups>
+        <group name="smile_elasticsuite_thesaurus">
+            <label translate="true">Manage Elasticsuite Thesaurus</label>
```

Comment:
Please add i18n files for en_US and fr_FR.
Translation for fr_FR is "Manage Elasticsuite Thesaurus","Gérer les Thesaurus Elasticsuite"